otx/algorithms/common/adapters/mmcv/utils.py

```diff
@@ -115,3 +115,12 @@ def prepare_work_dir(config: Union[Config, ConfigDict]) -> str:
         config.runner.meta = ConfigDict()
     config.runner.meta.exp_name = f"train_round_{len(checkpoint_dirs)}"
     return train_round_checkpoint_dir
+
+
+@check_input_parameters_type()
+def get_data_cfg(config: Union[Config, ConfigDict], subset: str = "train") -> Config:
```

Comment:
This also exists on the detection side. If it is a function used in action, how about moving it to `common`?

Reply:
Fair point, but I want this PR do not handle any other tasks. So I added TODO flag